### DISCUSSION TITLE
about the sponsorship

### DISCUSSION THREAD

**Discussion by capfredf** [emacs-helm/helm#2581] [General]
Hi Thierry,

I was wondering if you consider any approach other than paypal so we can sponsor this project? I just personally prefer not to give Paypal a fee.

**Comment by thierryvolpiatto** [comment]
Fred Fu ***@***.***> writes:

> Hi Thierry,
>
> I was wondering if you consider any approach other than paypal so we
> can sponsor this project?

For now I have nothing suitable.

> I just personally prefer not to give Paypal a fee.

Agree, I have removed all links to PayPal, they have actually blocked my
account for no good reasons, a very bad company I would recommend to
nobody.

> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.*Message ID: ***@***.***>


-- 
Thierry
